src/main/resources/data-config-sample.json

```diff
@@ -78,12 +79,14 @@
                 "type": "textfield",
                 "label": "Textfield Test",
                 "json-expose": true,
+                "id": "417671d1-8923-41eb-a315-cc5c2c05bf75",
```

Comment:
Are `id` now required?  Or is this optional, and needed for the UI?

Reply:
yeah, ID's are optional for the command line, it's a needed feature for the UI in order to re-order the fields on demand.